test/test-clusters/src/main/java/org/elasticsearch/test/cluster/local/LocalSpecBuilder.java

```diff
@@ -155,4 +156,10 @@ interface LocalSpecBuilder<T extends LocalSpecBuilder<?>> {
      * Adds an additional command line argument to node JVM arguments.
      */
     T jvmArg(String arg);
+
+    /**
+     * Register a function to compute config directory based on the node name. The default config directory
```

Comment:
I think we need to update this javadoc now.

Reply:
Yep. updated in d8bb5cd4b358efa8220b079803d31c6aa9ccdc14